src/arrayMethodJoin.js

```diff
@@ -5,8 +5,21 @@
  */
 function applyCustomJoin() {
   [].__proto__.join2 = function(separator) {
```

Comment:
@DidenkoDenis306 is right 👍 

this is how it works, when no separator is coming 
![image](https://github.com/mate-academy/js_array-method-join/assets/46566190/b5c9702c-f4ec-4a7f-aadb-cf56ddddfbaf)